src/components/Tokens/TokenDetails/TransactionsTable.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { screen } from '@testing-library/react'
+import { ChainId, Token } from '@uniswap/sdk-core'
+import { render } from 'test-utils/render'
+
+import { TransactionsTable } from './TransactionsTable'
+
+describe('TransactionsTable', () => {
+  const mockToken = new Token(
+    1,
+    '0x72e4f9f808c49a2a61de9c5896298920dc4eeea9',
+    18,
+    'BITCOIN',
+    'HarryPotterObamaSonic10Inu'
+  )
+
+  it('displays swap transaction', () => {
+    window.innerWidth = 1280
```

Comment:
does this need to be set? Wouldn't the elements still be in the document since you can scroll?